src/Symfony/Component/Console/Tests/Command/InvokableCommandTest.php

```diff
@@ -35,8 +35,14 @@ public function testCommandInputArgumentDefinition()
             #[Argument] ?string $firstName,
             #[Argument] string $lastName = '',
             #[Argument(description: 'Short argument description')] string $bio = '',
+            // Declare callback in static context, even when method is NOT static.
+            // PHP doesn't allow to use `$this` here, and callback is later modified
+            // on-the-fly to be called on instance instead: https://github.com/symfony/symfony/blob/7.3/src/Symfony/Component/Console/Attribute/Argument.php#L87
```

Comment:
we don't add links to github in the codebase
```suggestion
            // on-the-fly to be called on instance instead
```

Reply:
ack

let me apply of course,
i wonder if you have a practice how to provide such reference? it was really hidden knowledge, when looking at spec-via-tests

Reply:
you'll find references using eg `@see TheClass` in phpdoc

Reply:
tbh, myself, even with `@see Argument::tryFrom()`, i wouldn't intuitively find this behaviour, as it's not documented in that class/code and condition to apply this transformation is combined of 6 conditions 😅 

Perhaps i'm too far away from this, unexpected to me, behaviour.
Perhaps, OK to add some in-code docs for this transofmation? (added last commit, feel free to revert)